cmd/aro/operator.go

```diff
@@ -188,7 +188,7 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 			return fmt.Errorf("unable to create controller MachineSet: %v", err)
 		}
 		if err = (imageconfig.NewReconciler(
-			log.WithField("controller", controllers.ImageConfigControllerName),
+			controllers.ImageConfigControllerName),
```

Comment:
doesn't compile, `controllers.ImageConfigControllerName),` leftover from the logrus instantiation.
```suggestion
```